modules/ai/assets/js/editor/components/prompt-history/parts/modal-empty.js

```diff
@@ -0,0 +1,31 @@
+import { Stack, Typography } from '@elementor/ui';
+import { HISTORY_TYPES, TRANSLATED_HISTORY_TYPES, TRANSLATED_HISTORY_TYPES_FALLBACK } from '../history-types';
+
+const getTranslatedPromptType = ( promptType ) => {
+	if ( TRANSLATED_HISTORY_TYPES[ promptType ] ) {
+		return TRANSLATED_HISTORY_TYPES[ promptType ];
+	}
+
+	return TRANSLATED_HISTORY_TYPES_FALLBACK;
+};
+
+const PromptHistoryEmpty = ( { promptType } ) => {
+	return (
+		<Stack justifyContent="center" sx={ { height: '100%', textAlign: 'center' } } data-testid="e-ph-empty">
+			<Typography variant="h6">{ __( 'Prepare to be amazed', 'elementor' ) }</Typography>
+			<Typography variant="body1">
+				{ sprintf(
+					// Translators: %s: History type.
```

Comment:
Do you need this comment?

Reply:
Yes, otherwise eslint highlights it in red.